src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
A `Context` property is a better idea than accessing an internal field. I have made this change in the interest of improving code quality. However, this does not solve the problem if `signedXml` itself is null, as in the case above.